### DISCUSSION TITLE
Remove the null in RouterOutputs type

### DISCUSSION THREAD

**Discussion by AlexisKenAlvarez** [trpc/trpc#5577] [Q&A] [answered]
I want to remove the property of null in my type

`user: RouterOutputs["user"]["getSession"];`

![image](https://github.com/trpc/trpc/assets/80466376/4d4554d9-3848-4e6f-86c1-3b958c3c404c)
I just want to get User instead of User | null, how do i tell it that it will never be null?

**Comment by KATT** [comment, marked as answer]
`NonNullable<x>`

**Reply by AlexisKenAlvarez** [reply]
Thank you so much!
